ctagsdotd/vbnet.ctags

```diff
@@ -0,0 +1,12 @@
+# See additional-language.ctags for maintaining this file
+--langdef=vbnet
+--langmap=vbnet:.vb
+--regex-vbnet=/^(?i)\s*(?:(?:Public|Private|Protected|Friend|Overloads|Overrides|Overridable|NotOverridable|MustOverride|Shared|Shadows|Async|Iterator)\s+)*\s*(?:Function|Sub)\s+([A-Z_0-9\[\]]*)\(.*?\)\s*(?:As\s+[\w.]+)?/\1/m,method/
+--regex-vbnet=/^(?i)\s*(?:(?:Default|Public|Private|Protected|Friend|Overloads|Overrides|Overridable|NotOverridable|MustOverride|Shared|Shadows|WriteOnly|ReadOnly|Iterator)\s+)*\s*(?:Property)\s+([A-Z_0-9\[\]]*)\s+As\s+/\1/p,property/
```

Comment:
Should this regex have an extra check for `Let`/`Set`/`Get`?

Looking at one of the code examples here:

https://learn.microsoft.com/en-us/office/vba/language/concepts/getting-started/writing-a-property-procedure

```vb
Property Let Names(intX As Integer, intY As Integer, varZ As Variant) 
 ' Statement here. 
End Property 
 
Property Get Names(intX As Integer, intY As Integer) As Variant 
 ' Statement here. 
End Property 
```

This regex seems like it'll misidentify `Let`/`Get`/`Set` as the name.

Reply:
The link you provided is for [VBA](https://en.wikipedia.org/wiki/Visual_Basic_for_Applications#:~:text=As%20its%20name%20suggests%2C%20VBA,from%20another%20using%20OLE%20Automation.) (Visual Basic for Applications).  While this implementation is supposed to add support for [Visual Basic (.NET)](https://en.wikipedia.org/wiki/Visual_Basic_(.NET)).
My source for the implementation was: https://learn.microsoft.com/en-us/dotnet/visual-basic/language-reference/statements/property-statement